fail when make , Ubuntu 16.04 x64, Qt 5.8

complie GammaRay with command:

cmake ..
make

cmake finished, but make failed

it says:

```
[ 80%] Built target gammaray_actioninspector_ui_plugin
[ 80%] Automatic moc for target gammaray_actioninspector_plugin
[ 80%] Built target gammaray_actioninspector_plugin_automoc
[ 80%] Built target gammaray_actioninspector_plugin
[ 80%] Automatic moc for target gammaray_widget_export_actions
[ 80%] Built target gammaray_widget_export_actions_automoc
[ 80%] Built target gammaray_widget_export_actions
[ 81%] Automatic moc for target gammaray_widgetinspector_plugin
[ 81%] Built target gammaray_widgetinspector_plugin_automoc
Scanning dependencies of target gammaray_widgetinspector_plugin
[ 81%] Building CXX object plugins/widgetinspector/CMakeFiles/gammaray_widgetinspector_plugin.dir/gammaray_widgetinspector_plugin_automoc.cpp.o
[ 81%] Linking CXX shared module ../../lib/gammaray/2.8/qt5_8-x86_64/gammaray_widgetinspector_plugin.so
[ 82%] Built target gammaray_widgetinspector_plugin
[ 82%] Automatic moc for target gammaray_widgetinspector_ui_plugin
[ 82%] Built target gammaray_widgetinspector_ui_plugin_automoc
[ 83%] Built target gammaray_widgetinspector_ui_plugin
[ 83%] Generating WidgetDelegate.qml.qmllint
qmllint: could not exec '/usr/lib/x86_64-linux-gnu/qt4/bin/qmllint': No such file or directory
plugins/widgetinspector/assets/qml/CMakeFiles/WidgetDelegate.qml_qmllint.dir/build.make:60: recipe for target 'plugins/widgetinspector/assets/qml/WidgetDelegate.qml.qmllint' failed
make[2]: *** [plugins/widgetinspector/assets/qml/WidgetDelegate.qml.qmllint] Error 1
CMakeFiles/Makefile2:8277: recipe for target 'plugins/widgetinspector/assets/qml/CMakeFiles/WidgetDelegate.qml_qmllint.dir/all' failed
make[1]: *** [plugins/widgetinspector/assets/qml/CMakeFiles/WidgetDelegate.qml_qmllint.dir/all] Error 2
Makefile:160: recipe for target 'all' failed
make: *** [all] Error 2
```

Hi i have installed all this packet but i have this error now : 

`[ 98%] Generating Slider.qml.qmllint
/usr/lib/x86_64-linux-gnu/qt5/bin/qmllint: error while loading shared libraries: libicui18n.so.56: cannot open shared object file: No such file or directory
examples/quick-batching/CMakeFiles/Slider.qml_qmllint.dir/build.make:60 : la recette pour la cible « examples/quick-batching/Slider.qml.qmllint » a échouée
make[2]: *** [examples/quick-batching/Slider.qml.qmllint] Erreur 127
CMakeFiles/Makefile2:10101 : la recette pour la cible « examples/quick-batching/CMakeFiles/Slider.qml_qmllint.dir/all » a échouée
make[1]: *** [examples/quick-batching/CMakeFiles/Slider.qml_qmllint.dir/all] Erreur 2
Makefile:160 : la recette pour la cible « all » a échouée
make: *** [all] Erreur 2
`

--- a/cmake/FindQmlLint.cmake
+++ b/cmake/FindQmlLint.cmake
@@ -28,7 +28,18 @@
 
 find_program(QMLLINT_EXECUTABLE qmllint)
 if (QMLLINT_EXECUTABLE AND NOT Qt5Core_VERSION VERSION_LESS 5.4)
-  set(QmlLint_FOUND TRUE)
+  if(NOT QMLLINT_IS_WORKING)
+    # Try to fix common problems on Debian-based distros -- they provide /usr/bin/qmllint, which is a symlink to
+    # /usr/lib/x86_64-linux-gnu/qt4/bin/qmllint (or the Qt5 version of it). The actual executable is part of different
+    # package, so might not even be installed => double-check whether qmllint is working by executing it
+    execute_process(COMMAND ${QMLLINT_EXECUTABLE} --version RESULT_VARIABLE _qmllint_result OUTPUT_QUIET ERROR_QUIET)
+    if (_qmllint_result EQUAL 0)
+      set(QMLLINT_IS_WORKING TRUE CACHE BOOL "Whether the found qmllint executable is actually usable" FORCE)
+    endif()
+  endif()
+  if(QMLLINT_IS_WORKING)
+    set(QmlLint_FOUND TRUE)
+  endif()
 endif()
 
 # validate a list of qml files

